diff -up ./src/version.h.update ./src/version.h
--- ./src/version.h.update	2016-02-17 15:38:30.464094042 -0500
+++ ./src/version.h	2016-02-17 15:38:38.235314016 -0500
@@ -42,6 +42,6 @@
 #define CSCOPE_VERSION_H
 
 #define	FILEVERSION	15	/* Initial Open Source and Linux Port */
-#define	FIXVERSION	".7a"	/* feature and bug fix version */
+#define	FIXVERSION	".8"	/* feature and bug fix version */
 
 #endif /* CSCOPE_VERSION_H */
