diff -up ./src/find.c.nullfix ./src/find.c
--- ./src/find.c.nullfix	2016-01-07 10:35:59.258100168 -0500
+++ ./src/find.c	2016-01-07 10:37:07.930081508 -0500
@@ -48,8 +48,6 @@
 #endif
 #include <regex.h>
 
-static char const rcsid[] = "$Id: find.c,v 1.20 2006/08/20 19:15:54 broeker Exp $";
-
 /* most of these functions have been optimized so their innermost loops have
  * only one test for the desired character by putting the char and 
  * an end-of-block marker (\0) at the end of the disk block buffer.
@@ -1199,7 +1197,9 @@ getposting(void)
 static void
 putpostingref(POSTING *p, char *pat)
 {
-	static char	function[PATLEN + 1];	/* function name */
+	// initialize function to "unknown" so that the first line of temp1
+	// is properly formed if symbol matches a header file entry first time
+	static char function[PATLEN + 1] = "unknown";/* function name */
 
 	if (p->fcnoffset == 0) {
 		if (p->type == FCNDEF) { /* need to find the function name */
